Updated changelog and version (#98)
newton-physics/mujoco-usd-converter: CHANGELOG.md

# 0.4.0rc1

## Features

- Author `NewtonJointAPI`
  - Direct value authoring for `newton:armature`, `newton:friction`, and `newton:damping`
    - The equivalent `mjc:armature`, `mjc:frictionloss`, and `mjc:damping` are still authored as well, for applications not yet parsing `NewtonJointAPI`

>   - Converted value authroing for `newton:limitStiffness/damping` from `mjc:solreflimit` for limited joints, including angular per-degree scaling.

Reviewer: ```suggestion
  - Converted value authoring for `newton:limitStiffness/damping` from `mjc:solreflimit` for limited joints, including angular per-degree scaling.
```

- Author `newton:jointsAddMobility = true` on all articulation roots, reflecting the intent of MuJoCo's joint description.

## Breaking Changes

- Stopped authoring MJC schema aliases which have been deprecated in MuJoCo 3.10
 - Affects some scene options, collision margin/gap, mesh max hull vertices, material rolling/torsional friction, and equality mimic metadata

>  - All affected attributes have Newton equivalants which are now parsed in both MuJoCo and Newton.

Reviewer: ```suggestion
  - Affects some scene options, collision margin/gap, mesh max hull vertices, material rolling/torsional friction, and equality mimic metadata
  - All affected attributes have Newton equivalents which are now parsed in both MuJoCo and Newton.
```

- Mapped MuJoCo `margin` directly to `newton:contactMargin` and `gap` to `newton:contactGap`, matching the MuJoCo 3.10 margin/gap semantics

> ## Fixed

Reviewer: ```suggestion
## Fixes
```
